dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/V2CollectionController.java

```diff
@@ -24,29 +28,130 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@Api(description = "The collection controller V2", name = "Collection controller V2")
+@Tag(name = "V2CollectionController", description = "The CollectionController in legacy version V2")
```

Comment:
Können wir uns nicht eins der beiden Attribute sparen?

Reply:
In dem Fall nicht, weil die die Description hier etwas aussagekräftiger ist, als der Name